code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -161,13 +163,28 @@
 	if (get_dist(src, user) <= 1)
 		checktime()
 
+/obj/item/clothing/gloves/watch/verb/changeformat()
+	set category = "Object"
+	set name = "Change Watch Format"
+	set src in usr
+
+	use_military_format = !use_military_format
+	to_chat(usr, "You toggle the format of your watch. Now it will display in [use_military_format ? "24" : "12"] hour time.")
+
 /obj/item/clothing/gloves/watch/verb/checktime()
 	set category = "Object"
 	set name = "Check Time"
 	set src in usr
 
 	if(wired && !clipped)
-		to_chat(usr, "You check your watch, spotting a digital collection of numbers reading '[worldtime2text()]'. Today's date is '[time2text(world.time, "Month DD")]. [game_year]'.")
+		if(use_military_format)
+			to_chat(usr, "You check your watch, spotting a digital collection of numbers reading '[worldtime2text()]'. Today's date is '[time2text(world.time, "Month DD")]. [game_year]'.")
```

Comment:
I'd consider saving just the clock text to a separate variable, then using a single `to_chat` to output it for readability / ease of changing the message.

Reply:
I might go back and make that change. For now I'm going to commit the suggestions.